lib/puppet/provider/zabbix.rb

```diff
@@ -11,7 +11,9 @@ def self.create_connection(zabbix_url, zabbix_user, zabbix_pass, apache_use_ssl)
     zbx = ZabbixApi.connect(
       url: "#{protocol}://#{zabbix_url}/api_jsonrpc.php",
       user: zabbix_user,
-      password: zabbix_pass
+      password: zabbix_pass,
+      http_user: zabbix_user,
```

Comment:
what happens when those variables are empty?

Reply:
They never are empty since they re use the same regular Zabbix API credentials.
And in a setup without HTTP auth, the HTTP Auth header is just ignored by Apache.

Reply:
okay, thanks for the explanation.